kubeflow/env.sh

```diff
@@ -1,12 +1,39 @@
-export KF_NAME=<kubeflow-cluster-name>
+# 1. Edit <placeholders>.
+# 2. Other env vars are configurable, but with default values set below.
+
+# Your Google Cloud project ID
 export KF_PROJECT=<gcp-project-id>
-export KF_PROJECT_NUMBER=$(gcloud projects describe ${KF_PROJECT} --format='value(projectNumber)')
-export KF_DIR=<kubeflow-download-path>
-export MGMT_NAME=<management-cluster-name>
-export MGMTCTXT=${MGMT_NAME}
-export LOCATION=<zone>
+# You can get your project number by running this command:
+# gcloud projects describe --format='value(projectNumber)' "${KF_PROJECT}"
+export KF_PROJECT_NUMBER=<gcp-project-number>
+# Kubeflow Admin email address
+# Example: admin@gmail.com
 export ADMIN_EMAIL=<administrator-full-email-address>
+# The management cluster name you have already deployed following:
+# https://www.kubeflow.org/docs/distributions/gke/deploy/management-setup/
+export MGMT_NAME=<management-cluster-name>
+# Management cluster kubectl context name, defaults to ${MGMT_NAME} if you
+# followed the above guide.
+export MGMTCTXT="${MGMT_NAME}"
 
-export ASM_LABEL=asm-192-1
+# The following env vars are also configurable.
 
+# KF_NAME is your Kubeflow cluster name.
+# It should satisfy the following conditions:
+# * is unique within your project.
+# * alphanumeric characters and hyphen "-" only.
+# * starts with a letter.
+# * ends with an alphanumeric character.
```

Comment:
Yes, I don't remember the limit off the top of my head. We can add it later (non-release blocker)

Reply:
If I remember correctly, it is `no more than 25 characters`, I think it is a calculated result because this name is used as a prefix for longer variables. But I don't know the actual formula for getting number 25.